pubtools/_quay/remove_repo.py

```diff
@@ -135,9 +136,18 @@ def remove_repositories(repositories: str, settings: dict[str, Any]) -> None:
             continue
         outdated_manifests.append((mad.digest, tag, repo))
 
+    signer_settings = {}
+    for k, v in settings.items():
+        if k == "quay_org":
+            signer_settings["quay_namespace"] = v
```

Comment:
Can you give an explanation in the commit message of this change and why it was necessary?

Reply:
sure